frontend/src/assets/css/loader.scss

```diff
@@ -0,0 +1,82 @@
+$loader-bg-color: #ffffff80;
+$loader-color: #cceeff;
+
+.ld-container {
+  position: fixed;
```

Comment:
let's make it via position absolute, we may use loader inside other components not only for the whole app